app/gui/src/project-view/components/ComponentBrowser/ComponentTypeLabel.vue

```diff
@@ -41,14 +41,10 @@ const label = computed(() => {
 </script>
 
 <template>
-  <div v-if="label" :data-testid="props.testId" class="componentEditorLabel">
+  <div v-if="label" :data-testid="props.testId" class="typeLabelBackground">
```

Comment:
Usually within a component we apply a class with the same name as the component to the root node, and use that for all root styling (although this was already not followed by this component, prior to this PR). Since Vue's style scoping allows a component's root node's classes to be styled by the parent component, this serves to avoid clashes that could occur when unexpected class names are exposed to the parent.